common/utils/RandomGenUtils.php

```diff
@@ -20,10 +20,10 @@ public static function produceRandomBytes($length)
         try {
             $randomBytes = random_bytes($length);
         } catch (Error $e) {
-            error_log('OpenEMR Error : Encryption is not working because of random_bytes() Error: ' . $e->getMessage());
+            error_log('OpenEMR Error : Encryption is not working because of random_bytes() Error: ' . errorLogEscape($e->getMessage()));
             return '';
         } catch (Exception $e) {
-            error_log('OpenEMR Error : Encryption is not working because of random_bytes() Exception: ' . $e->getMessage());
+            error_log('OpenEMR Error : Encryption is not working because of random_bytes() Exception: ' . errorLogEscape($e->getMessage());
```

Comment:
need another parenthesis at end of this line (otherwise this will throw a parsing error)

Reply:
Currently editing this line and the other two mentioned, I went ahead and submitted changes before scrolling all the way down here!